README.md

```diff
@@ -135,6 +135,7 @@ Then the test can call the module, and the module will do the HTTP requests.
 When you setup an interceptor for a URL and that interceptor is used, it is removed from the interceptor list.
 This means that you can intercept 2 or more calls to the same URL and return different things on each of them.
 It also means that you must setup one interceptor for each request you are going to have, otherwise nock will throw an error because that URL was not present in the interceptor list.
+But if you whant use .persist() and then nock will always intercept the requests, read more in * [.persist()](#persist)
```

Comment:
Could you rephrase the sentence to 

> If you don’t want interceptors to be removed as they are used, you can use the [.persist()](#persist) method.

Or add me as collaborator to your fork and I can fix it up for you :)